src/models/classes/order.js

```diff
@@ -0,0 +1,23 @@
+class Order
+{
+    /**
+     * Represents an order (only for customer for now)
+     *
+     * @param products
+     */
+    constructor(products)
+    {
+        this.products = products;
+        // this.orderCost = this.orderCost();
```

Comment:
This one line can go, as you've already seen, functions can be called from within Mustache! Quite useful :).